README.md

```
@@ -11,7 +11,7 @@ This project requires AD4M! The easiest way to have an AD4M runtime operating, i
 ### Compiles and hot-reloads for development
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)